test/unit/math/prim/fun/linspaced_int_array_test.cpp

```diff
@@ -0,0 +1,31 @@
+#include <stan/math/prim.hpp>
+#include <test/unit/util.hpp>
+#include <gtest/gtest.h>
+#include <limits>
+#include <vector>
+
+TEST(MathFunctions, linspaced_int_array) {
+  using stan::math::linspaced_int_array;
+  EXPECT_STD_VECTOR_EQ(linspaced_int_array(0, 1, 5), std::vector<int>({}));
+  EXPECT_STD_VECTOR_EQ(linspaced_int_array(1, 1, 5), std::vector<int>({5}));
```

Comment:
Huh, that's surprising. I'd have expected the output to be `1`. `5` seems strange. Is that really what Eigen does?

Reply:
Actually I guess we should do whatever the already existing linspaced stuff does.

Reply:
Yeah that is why its done that way.